semgrep_output_v1.atd

```diff
@@ -3309,6 +3309,19 @@ type symbol_analysis_upload_response = {
 (* Symbol analysis *)
 (* ----------------------------- *)
 
+(* The input needed to run symbol analysis for a project. *)
+type symbol_analysis_params = {
+    (* root path of the subproject—usually the directory of the manifest file *)
+    root_path: fpath;
+
+    (* the language of the files to analyze; this will ideally be
+       optional/ignored in the future *)
+    lang : string;
```

Comment:
Should we make this optional today? We can keep passing it in, but we will have to potentially modify any ocaml code that relies on it existing.